src/menus/menu-divider.jsx

```diff
@@ -1,59 +1,36 @@
 import React from 'react';
-import StylePropable from '../mixins/style-propable';
-import ListDivider from '../lists/list-divider';
-import DefaultRawTheme from '../styles/raw-themes/light-raw-theme';
-import ThemeManager from '../styles/theme-manager';
+import Divider from '../divider';
+import Styles from '../utils/styles';
```

Comment:
I don't mind renaming, I initially called it Styles because that's the name it was imported under in `style-propable.js`. But I understand that is going to be removed soon.

@oliviertassinari , @subjectix ,
If we were to rename this, what do you both prefer?
`utilsStyles`
`styleUtil`
`styleUtils`